### PR TITLE
feat(telecom.telephony): add reset password white label manager

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-8245 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I added a section for reset password of white label manager

## Related
:flags:  Translations:

- [x] TRANS-44606

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.controller.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+export default class TelecomTelephonyBillingAccountAdministrationWhiteLabelManagerCtrl {
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.controller.js:1]: **suggestion:** The name is too big, is it possible to shorten it?

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<section>
+    <oui-header
+        data-heading="{{ :: 'telephony_white_label_manager_title' | translate }}"
+        data-description="{{ :: 'telephony_white_label_manager_descrition' | translate }}"
+    >
+        <oui-back-button
+            data-href="{{ OptionsGroupCtrl.$state.href('telecom.telephony.billingAccount.administration') }}"
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html:7]: **nitpick:** Maintain the URL in the `$ctrl` associated with this template.

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    <p
+        class="mt-1"
+        data-translate="telephony_white_label_manager_reset_password_description"
+    ></p>
+
+    <p>
+        <strong
+            data-translate="telephony_white_label_manager_reset_password_warning"
+        ></strong>
+    </p>
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html:28]: **question:** Why not use `strong`?
```suggestion
        <strong
            data-translate="telephony_white_label_manager_reset_password_warning"
        ></strong>
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html (hunk 2 above)
Comment (ghyenne) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html]: I fixed.

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html (hunk 3 above)
Comment (ghyenne) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html]: I fixed.

**Review by anooparveti — DISMISSED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "telephony_white_label_manager_title": "Manager marque blanche",
+  "telephony_white_label_manager_descrition": "Le manager marque blanche vous offre une interface personnalisable aux couleurs de votre société pour gérer les services Telecom OVHcloud.",
+  "telephony_white_label_manager_checkbox_label": "Je souhaite réinitialiser le mot de passe de mon manager marque blanche.",
```
Comment (mohammed-zahaf) [inline packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/translations/Messages_fr_FR.json:4]: i guess it's better with a upper capital lettrer, non ?
```suggestion
  "telephony_white_label_manager_checkbox_label": "Je souhaite réinitialiser le mot de passe de mon Manager marque blanche.",
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.service.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+export default class TelecomTelephonyWhiteLabelManagerService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  resellerPanelResetPassword() {
+    return this.$http.post('/telephony/resellerPanel/generatePassword');
```
Comment (mohammed-zahaf) [inline packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.service.js:8]:
```suggestion
    return this.$http.post('/telephony/resellerPanel/generatePassword').then(({data}) => data);
```

**Review by mohammed-zahaf — APPROVED** [review]
Good job :)

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.service.js (hunk 5 above)
Comment (ghyenne) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.service.js]: I fixed.

Comment (ghyenne) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.service.js]: I fixed.

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/translations/Messages_fr_FR.json (hunk 4 above)
Comment (ghyenne) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/translations/Messages_fr_FR.json:4]: I fixed.

Comment (ghyenne) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/translations/Messages_fr_FR.json:4]: I fixed.

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by vks-ovh — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+  "telephony_white_label_manager_title": "Manager marque blanche",
+  "telephony_white_label_manager_descrition": "Le Manager marque blanche vous offre une interface personnalisable aux couleurs de votre société pour gérer les services Telecom OVHcloud.",
+  "telephony_white_label_manager_checkbox_label": "Je souhaite réinitialiser le mot de passe de mon Manager marque blanche.",
+  "telephony_white_label_manager_reset_password_title": "Réinitialisation du mot de passe",
+  "telephony_white_label_manager_reset_password_description": "Cette procédure réinitialisera le mot de passe de votre Manager marque blanche.",
+  "telephony_white_label_manager_reset_password_warning": "Cette action entraine la désactivation de l'ensemble des comptes utilisateurs de votre manager et nécessite que vous reconfiguriez leur mots de passes.",
```
Comment (mohammed-zahaf) [inline packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/translations/Messages_fr_FR.json:7]: Missed one :) `Ctrl+F/Ctrl+R`
```suggestion
  "telephony_white_label_manager_reset_password_warning": "Cette action entraine la désactivation de l'ensemble des comptes utilisateurs de votre Manager et nécessite que vous reconfiguriez leur mots de passes.",
```

**Review by anooparveti — DISMISSED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by vks-ovh — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by sachinrameshn — DISMISSED** [review]

**Comment by antleblanc** [comment]
Pulling translations.

**Review by antleblanc — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**